plugins/nodes/src/js/pages/nodes/NodeDetailPage.js

```diff
@@ -128,6 +141,67 @@ class NodeDetailPage extends mixin(TabsMixin, StoreMixin) {
     );
   }
 
+  getActions() {
+    const hasMaintenance = findNestedPropertyInObject(
+      ConfigStore.get("config"),
+      "uiConfiguration.features.maintenance"
+    );
+
+    if (!hasMaintenance) {
+      return [];
+    }
+
+    const actions = [];
+    const { node } = this.props;
+    const status = Status.fromNode(node);
+
+    if (actionAllowed(StatusAction.DRAIN, status)) {
```

Comment:
Is there any possibility of using `NodeActionsDropdown` here? I see that the action shape is slightly different but thought I'd check since they both seem to use reactjs-components `Dropdown` under the hood

Reply:
Hi-- This was my original intent but it turns out that PageHeader has it's own actions rendering logic and API. I found it was better to duplicate the actions here instead of rewrite page header actions to look more like reactjs-dropdown